### PR TITLE
[New] Managed Services

### PR DESCRIPTION
**Pull request by wildmanonline**
This PR includes a rewrite of the Managed Services product documentation.

- Managed Services > [Overview](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/)
- Managed Services > [Get Started](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/get-started/)
- Managed Services > [Guides](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/guides/)
- Managed Services > Guides > [Create and Manage Service Monitors](1ab8.netlify.app/docs/products/services/managed/guides/service-monitors/)
- Managed Services > Guides > [Configure SSH Access](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/guides/ssh-access/)
- Managed Services > Guides > [Manage Credentials](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/guides/credentials/)
- Managed Services > Guides > [Manage Contacts](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/guides/contacts/)
- Managed Services > Guides > [Configure Firewall Rules to Allow Access from Linode Infrastructure](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/guides/allow-access-from-linode-infrastructure/)
- Managed Services > Guides > [Cancel Managed Services](https://deploy-preview-6229--nostalgic-ptolemy-b01ab8.netlify.app/docs/products/services/managed/guides/cancel/)

### CODE REVIEW

File: docs/products/services/managed/guides/contacts/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+description: "Learn how to add contacts and contact groups to Linode Managed Services."
+published: 2023-04-04
+authors: ["Linode"]
+---
+
+If a Service Monitor detects an issue with your system, it can be configured to send an email notification to the people on your team that need to be alerted. This contact information is stored as a *Contact* in Managed Services. Each Service Monitor can be attached to a *Contact Group*, each of which can contain one or more individual *Contacts*. Separate groups can be created for each different system or service.
+
+For example, you might add all of your database administrators to a group named *DBAs* and then link that group to a Service Monitor that's monitoring the state of your MySQL server. When an issue is detected, all of your database administrators are notified at the same time.
+
+You can optionally attach up to two phone numbers to your contacts, a primary number and a secondary number. This is used by our Support team to call a point of contact on your side to obtain more information when troubleshooting an issue.
```
Comment (andystevensname) [inline docs/products/services/managed/guides/contacts/index.md:13]: These are used

File: docs/products/services/managed/guides/credentials/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Add a Credential
+
+1. Navigate to the **Credentials** tab for Managed Services in the Cloud Manager. See [View Credentials](#view-credentials).
+
+1. Click the **Add Credentials** button.
+
+1. In the form that appears, enter the details for your credential. The following fields are available:
+
+    - **Label** (required): A descriptive name for the credential that tells the Linode Support Team what it should be used for. For example, if you are entering the MySQL `root` password, you might label it as `MySQL Root`.
```
Comment (andystevensname) [inline docs/products/services/managed/guides/credentials/index.md:33]: A descriptive name for the credential that explains to the Linode Support Team how the credential should be used.

File: docs/products/services/managed/guides/ssh-access/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+1. Edit the SSH authorized keys file for whichever user account you want the Support Team to use when logging in.
+
+    - For the root user:
+
+        ```command
+        sudo nano /root/.ssh/authorized_keys
+        ```
+
+        {{< note title="Note for using the root user" isCollapsible=true >}}
+        Also, if you followed the instructions in the [Securing Your Server](/docs/products/compute/compute-instances/guides/set-up-and-secure/) guide to disable root login via SSH, you need to reenable that feature. Follow the relevant instructions within the [Set Up and Secure](/docs/products/compute/compute-instances/guides/set-up-and-secure/#ssh-daemon-options) guide to edit the `sshd_config` file and reenable root login via SSH.
```
Comment (andystevensname) [inline docs/products/services/managed/guides/ssh-access/index.md:46]: Don't need also

**Review by andystevensname — APPROVED** [review]

File: docs/products/services/managed/_index.md [hunk 4]
```diff
@@ -6,38 +6,47 @@ tab_group_main:
...
 
-Managed Services can be added to any account and can be used to monitor services across [all regions](https://www.linode.com/global-infrastructure/).
+{{< note >}}
+Linode Managed applies to all Compute Instances on an account **except** for nodes created and implemented by the [Linode Kubernetes Engine (LKE)](https://www.linode.com/products/kubernetes/). All eligible nodes are billed at a rate of $100 per month for each Linode. If this service is not needed for all instances, a second account can be created to separate Managed Compute Instances from non-Managed ones. If needed, instances can be transferred by [opening up a Support ticket](/docs/products/platform/get-started/guides/support/#contacting-linode-support) from both accounts and requesting the transfer. You can also contact the support team to install cPanel, add a cPanel license, or transfer a cPanel license.
```
[the remark this answers is not in the document]
Comment (wildmanonline) [reply docs/products/services/managed/_index.md:17]: Oh heck yes. Good catch.

File: docs/products/services/managed/_index.md [hunk 5]
```diff
@@ -6,38 +6,47 @@ tab_group_main:
...
 
-Linode actively tracks uptime and responsiveness for every registered system and service. If a check fails, our experts take immediate steps to get your systems back online as quickly as possible.
+- **Included Services and Software:** The following services and software applications are included at no additional charge to Linode Managed customers
 
-### Metrics, Monitoring, and Protection
+    - [cPanel](https://cpanel.net/): A cPanel license is included for each Linode on your account. cPanel is typically licensed by the number of accounts within the cPanel installation, each account typically corresponding to a single website or group of similar websites. This license will automatically scale based on the number of cPanel accounts you've configured.
+    - [Backups](https://www.linode.com/products/backups/): Each Linode on your account will also be subscribed to our Backup service for no extra charge. This service automatically backs up each Linode every day and allows you to restore from the last daily backup, last week's backup, or the previous week's backup.
```
[the remark this answers is not in the document]
Comment (wildmanonline) [reply docs/products/services/managed/_index.md:27]: Clarified this a bit:

> - [Backups](/docs/products/storage/backups/): The Backup service is added to each Compute Instance on your account for no extra charge. This service automatically backs up the Compute Instance each day and allows you to restore from the most recent daily backup, weekly backup, and biweekly backup. See [Get Started with Backups](/docs/products/storage/backups/get-started/) for more information on backup restore points.

**Pull request merged**